### PR TITLE
DOC-493 - CENM files missing front matter 

### PR DESCRIPTION
**Pull request by NeilAndersonR3**

### CODE REVIEW

**Comment by NeilAndersonR3** [comment]
Old menu:
image

New menu:
image
image

File: content/en/platform/corda/1.3/cenm/config-shell.md [hunk 1]
```diff
@@ -2,7 +2,11 @@
 aliases:
 - /config-shell.htmls
 date: '2020-01-08T09:59:25Z'
-menu: []
+menu:
+  cenm-1-3:
+    identifier: cenm-1-3-config-shell-parameters
```
Comment (AnnaK-R3) [inline content/en/platform/corda/1.3/cenm/config-shell.md:7]:
```suggestion
    identifier: cenm-1-3-config-shell
```

File: content/en/platform/corda/1.3/cenm/config-shell.md [hunk 2]
```diff
@@ -2,7 +2,11 @@
...
+    parent: cenm-1-3-configuration
+    weight: 240
```
Comment (AnnaK-R3) [inline content/en/platform/corda/1.3/cenm/config-shell.md:9]: Why weight: 240? Should it be 250 to appear after config-pki-tool-parameters?

File: content/en/platform/corda/1.3/cenm/config-ssl.md [hunk 3]
```diff
@@ -2,7 +2,11 @@
 aliases:
 - /config-ssl.html
 date: '2020-01-08T09:59:25Z'
-menu: []
+menu:
+  cenm-1-3:
+    identifier: cenm-1-3-config-ssl-settings
```
Comment (AnnaK-R3) [inline content/en/platform/corda/1.3/cenm/config-ssl.md:7]:
```suggestion
    identifier: cenm-1-3-config-ssl
```

File: content/en/platform/corda/1.3/cenm/config-ssl.md [hunk 4]
```diff
@@ -2,7 +2,11 @@
...
+    parent: cenm-1-3-configuration
+    weight: 240
```
Comment (AnnaK-R3) [inline content/en/platform/corda/1.3/cenm/config-ssl.md:9]: Why weight: 240? Should it be 260 to appear after config-shell?

File: content/en/platform/corda/1.3/cenm/contract-whitelisting.md [hunk 5]
```diff
@@ -2,7 +2,11 @@
 aliases:
 - /contract-whitelisting.html
 date: '2020-01-08T09:59:25Z'
-menu: []
+menu:
+  cenm-1-3:
+    identifier: cenm-1-3-config-contract-whitelisting
```
Comment (AnnaK-R3) [inline content/en/platform/corda/1.3/cenm/contract-whitelisting.md:7]:
```suggestion
    identifier: cenm-1-3-contract-whitelisting
```

File: content/en/platform/corda/1.3/cenm/jira-setup.md [hunk 6]
```diff
@@ -2,7 +2,11 @@
 aliases:
 - /jira-setup.html
 date: '2020-01-08T09:59:25Z'
-menu: []
+menu:
+  cenm-1-3:
+    identifier: cenm-1-3-config-jira-setup
```
Comment (AnnaK-R3) [inline content/en/platform/corda/1.3/cenm/jira-setup.md:7]:
```suggestion
    identifier: cenm-1-3-jira-setup
```

File: content/en/platform/corda/1.3/cenm/setting-up-notary.md [hunk 7]
```diff
@@ -2,7 +2,11 @@
 aliases:
 - /setting-up-notary.html
 date: '2020-01-08T09:59:25Z'
-menu: []
+menu:
+  cenm-1-3:
+    identifier: cenm-1-3-notary-node
```
Comment (AnnaK-R3) [inline content/en/platform/corda/1.3/cenm/setting-up-notary.md:7]:
```suggestion
    identifier: cenm-1-3-setting-up-notary
```

**Review by yarivus — APPROVED** [review]
LGTM

**Comment by NeilAndersonR3** [comment]
Front matter updated for all CENM versions, preview site available here: https://neil.preview.docs.r3.com/en/platform/corda/1.3/cenm.html

**Review by AnnaK-R3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thx @NeilAndersonR3

**Pull request merged**